crypto/rand/drbg_ctr.c

```diff
@@ -287,7 +296,10 @@ __owur static int drbg_ctr_instantiate(RAND_DRBG *drbg,
 
     memset(ctr->K, 0, sizeof(ctr->K));
     memset(ctr->V, 0, sizeof(ctr->V));
-    if (!EVP_CipherInit_ex(ctr->ctx, ctr->cipher, NULL, ctr->K, NULL, 1))
+    if (!EVP_CipherInit_ex(ctr->ctx_ecb,
+                           ctr->cipher_ecb, NULL, ctr->K, NULL, 1)
+        || !EVP_CipherInit_ex(ctr->ctx_ctr,
+                              ctr->cipher_ctr, NULL, ctr->KX, NULL, 1))
```

Comment:
I feel here `ctr->ctx_ctr` key initialization is not needed. Because anyway after this `ctr_update()` initializes `ctr->ctx_ctr` with updated `ctx->K`